pkg/operation/botanist/component/vpnseedserver/vpn_seed_server.go

```diff
@@ -0,0 +1,645 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vpnseedserver
+
+import (
+	"context"
+	"fmt"
+
+	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
+	"github.com/gardener/gardener/pkg/operation/botanist/component"
+	kutil "github.com/gardener/gardener/pkg/utils/kubernetes"
+	"github.com/gardener/gardener/pkg/utils/managedresources"
+	"github.com/gogo/protobuf/types"
+
+	"istio.io/api/networking/v1beta1"
```

Comment:
```suggestion
	istionetworkingv1beta1 "istio.io/api/networking/v1beta1"
```

Btw, what is the difference compared to

```
	networkingv1beta1 "istio.io/client-go/pkg/apis/networking/v1beta1"
```
?

Reply:
The networkingv1beta1 packages ("istio.io/client-go/pkg/apis/networking/v1beta1") includes the outer struct definition, e.g. DestinationRule, Gateway and VirtualService. The istionetworkingv1beta1 package ("istio.io/api/networking/v1beta1") contains the sub-structure, which is unfortunately called the same, e.g. DestinationRule, Gateway and VirtualService.
However, to create the complete objects we need both, e.g. networkingv1beta1.DestinationRule{ Spec: istionetworkingv1beta1.DestinationRule { ... } }.

Reply:
The suggestion concerning the more explicit usage (v1beta1 vs. istionetworkingv1beta1) will be implemented.

Reply:
Thanks!